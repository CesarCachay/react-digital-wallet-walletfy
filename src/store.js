import { createStore, applyMiddleware, compose } from "redux";
import logger from "redux-logger";
import reducer from "./reducer";

const composer = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const store = createStore(reducer, composer(applyMiddleware(logger)));

export default store;
