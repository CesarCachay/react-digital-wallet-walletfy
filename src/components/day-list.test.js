import React from "react";
import { render, fireEvent } from "@testing-library/react";
import { Provider } from "react-redux";
import store from "../store";

import DayList from "./day-list";

test("DayList render as expenses", () => {
  const { asFragment } = render(
    <Provider store={store}>
      <DayList isExpense={true} />
    </Provider>
  );
  expect(asFragment()).toMatchSnapshot();
});

test("DayList render as incomes", () => {
  const { asFragment } = render(
    <Provider store={store}>
      <DayList isExpense={false} />
    </Provider>
  );
  expect(asFragment()).toMatchSnapshot();
});
