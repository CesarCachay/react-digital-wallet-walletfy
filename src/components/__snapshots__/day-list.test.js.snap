// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DayList render as expenses 1`] = `<DocumentFragment />`;

exports[`DayList render as incomes 1`] = `<DocumentFragment />`;
